js/therock.js

```diff
@@ -1251,6 +1253,93 @@ module.exports = class therock extends Exchange {
         return this.parseOrder (response);
     }
 
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name exmo#fetchOHLCV
+         * @description fetches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+         * @param {str} symbol unified symbol of the market to fetch OHLCV data for
+         * @param {str} timeframe the length of time each candle represents in minutes
+         * @param {int|undefined} since timestamp in ms of the earliest candle to fetch
+         * @param {int|undefined} limit the maximum amount of candles to fetch
+         * @param {dict} params extra parameters specific to the exmo api endpoint
+         * @returns {[[int]]} A list of candles ordered as timestamp, open, high, low, close, volume
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const periodInSeconds = this.numberToString (this.parseTimeframe (timeframe));
+        const period = Precise.stringDiv (periodInSeconds, '60');
```

Comment:
In case of timestamps and timedeltas integer math is fine:

```suggestion
        const periodInSeconds = this.parseTimeframe (timeframe);
        const periodInMinutes = periodInSeconds / 60;
```